force-app/main/default/classes/RecordManager.cls

```diff
@@ -0,0 +1,28 @@
+public with sharing class RecordManager {
+  public static List<Job_Application__c> getRecentRecruiterJAAndRelated() {
```

Comment:
```suggestion
  public static List<Job_Application__c> selectJobApplicationsWithReviewsCreatedLastWeek() {
```
It's quite long, but it's the best name for method.
Generally "get"/"select" syntax works and it depends on project code styles. However, I suggest use "get" as getter (for property) or when you get list of something (e.g. list od DTO objects, wrapper objects, instances of classes)